### ISSUE TITLE
<Replace this with actual title>

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1856]
## Problem

On the row audio player, the screen reader reads the bullet character between the 'by' line and the track length:
<img width="545" alt="Screen Shot 2022-09-22 at 8 42 06 PM" src="https://user-images.githubusercontent.com/15233243/191815781-00c03b4e-88d9-47bd-b25e-f9624240ad37.png">

## Description

Bullet character between “by {author}” and track length should have `aria-hidden=”true”` to avoid reading by screen readers.

## Additional context

Reported by Joe Dolson (https://make.wordpress.org/openverse/2022/09/07/frontend-release-v3-4-8-and-a-call-for-a11y-testing/#comment-95)

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by Kevan-Y** [comment]
Hi would like to try to resolve this

**Comment by obulat** [comment]
Sure, @Kevan-Y, I've assigned the issue to you.
